experimental/Schemes/src/ToricBlowups/constructors.jl

```diff
@@ -19,14 +19,13 @@ end
 
 
 @doc raw"""
-    blow_up(v::NormalToricVariety, exceptional_ray::AbstractVector{<:IntegerUnion}; coordinate_name::String)
+    blow_up(X::NormalToricVarietyType, primitive_vector::AbstractVector{<:IntegerUnion}; coordinate_name::Union{String, Nothing} = nothing)
+    -> ToricBlowupMorphism
 
-Blow up the toric variety by subdividing the fan of the variety with the
-provided exceptional ray. This function returns the corresponding morphism.
-
-Note that this ray must be a primitive element in the lattice Z^d, with
-d the dimension of the fan. In particular, it is currently impossible to
-blow up along a ray which corresponds to a non-Q-Cartier divisor.
+Star subdivide the fan $\Sigma$ of the toric variety $X$ along the
+primitive vector `primitive_vector` in the support of $\Sigma$ (Section
```

Comment:
This reads a bit clunky to me. Maybe something like this?
```suggestion
primitive vector `prim_v` in the support of $\Sigma$ (Section
```
or 
```suggestion
primitive vector `v` in the support of $\Sigma$ (Section
```
together with the corresponding changes in the function signature and implementation?

Reply:
Or maybe the following?
```suggestion
given primitive vector in the support of $\Sigma$ (Section
```

Reply:
This last variant would seem to align well with the documentation of `total_transform` and alike, for whichthe variable name is also not stated explicitly in the docstring.

Reply:
Changed